patches/chrome-browser-resources-bookmarks-shared_style.html.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/chrome/browser/resources/bookmarks/shared_style.html b/chrome/browser/resources/bookmarks/shared_style.html
+index b759e57dc179beafe12844b19f97a49bcc52ffe7..f95b23cf712488dcc8b108788b920d23b0858259 100644
+--- a/chrome/browser/resources/bookmarks/shared_style.html
++++ b/chrome/browser/resources/bookmarks/shared_style.html
+@@ -68,6 +68,10 @@
+     .folder-icon[open] {
+       content: url(chrome://theme/IDR_FOLDER_OPEN);
+     }
++    @media (prefers-color-scheme: dark) {
```

Comment:
@petemill Can we avoid this patch?

Reply:
Yes, the patch can be avoided (I'm 95% sure).

Problem is we don't have bookmarks frontend resources setup for overrides like we do for settings resources with `brave_settings_overrides.{html, js}`. [This outdated PR sets it up for the _history_ page](https://github.com/brave/brave-core/pull/2911/files), but for _bookmarks_ there would be no changes to `optimize_webui.py` needed, since _history_ unpaks resources just like _settings_ does, so we can unpak brave history resources to the same location, just like we do for settings (via `brave/browser/resources/settings/BUILD.gn`.

Once that's setup, we'd include this in `brave_history_overrides.html`:

```html
<dom-module id="brave-override-style-settings-ui">
  <template>
    <style>
      @media (prefers-color-scheme: dark) {
         .folder-icon[open] { content: url(chrome://theme/IDR_BRAVE_BOOKMARK_FOLDER_OPEN_WHITE); }
        .folder-icon { content: url(chrome://theme/IDR_BRAVE_BOOKMARK_FOLDER_CLOSED_WHITE); }
      }
    </style>
  </template>
</dom-module>
```

Reply:
@petemill I tried https://github.com/brave/brave-core/pull/5690/commits/63306b0d5c81af8792e3cd5a962651eeb2c8c621. but it seems not working.
Can you check this commit? maybe I need to do something more?

Reply:
@simonhong I think this will all change with C84; we might wait for that to land first 😄